dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/HibernateTrackedEntityStore.java

```diff
@@ -95,6 +95,7 @@
 class HibernateTrackedEntityStore extends SoftDeleteHibernateObjectStore<TrackedEntity>
     implements TrackedEntityStore {
 
+  private static final String DEFAULT_ORDER = "TE.trackedentityid ASC";
```

Comment:
Didn't we agree that default order would be `DESC`?

Reply:
Yes 😄 I am first adding tests and trying to change as little as needed. The next PRs will then implement whats missing.